add docs for custom action and swipes (#12)
tinkoff-mobile-tech/TiRecycler: README.md

}
```

# How to use swipe to dismiss

First of all, you need to create a callback helper class

```kotlin
ItemDismissTouchHelperCallback(
    context,
    dismissibleViewType = youViewType,
    icon = yourDrawable,
    dismissBackgroundColor = Color.RED,
    normalBackgroundColor = Color.WHITE,
    elevation = 1f,
    cornerRadius = 2f,
    drawableOffsetInDp = 24,
)
```

and add it to `itemDismissCallbacks` in you builder function for TiRecycler

```kotlin

> TiRecyclerCoroutines<ViewTyped>(recyclerView, holderFactory) { //for RX use similar function builder 

Reviewer: pls, write comment above the code, not inside

Author: done

    //...
    itemDismissCallbacks += ItemDismissTouchHelperCallback()
}
```

and after it, you can collect events via `recycler.swipeToDismiss` method

# How to use custom actions

> You need to implement `TiRecyclerCustomActionFlow/TiRecyclerCustomActionObservable` after it should add to map of you
> holder factory

Reviewer: ```suggestion
You need:
* implement your own action with extends from `TiRecyclerCustomActionFlow/TiRecyclerCustomActionObservable` 
* add it to map in your holder factory
```

Author: done
